ludwig/features/image_feature.py

```diff
@@ -250,12 +258,24 @@ def add_feature_data(
         if num_images == 0:
             raise ValueError('There are no images in the dataset provided.')
 
-        first_path = next(iter(input_df[feature[COLUMN]]))
+        first_img_source = next(iter(input_df[feature[COLUMN]]))
```

Comment:
We may call this `first_img_entry` for consistency

Reply:
made change